Persephone/Layouts/AlbumViewLayout.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  AlbumViewLayout.swift
+//  Persephone
+//
+//  Created by Daniel Barber on 2019/2/18.
+//  Copyright © 2019 Dan Barber. All rights reserved.
+//
+
+import Cocoa
+
+class AlbumViewLayout: NSCollectionViewFlowLayout {
+  let maxItemWidth: CGFloat = 180
+
+  required init?(coder aDecoder: NSCoder) {
+    super.init()
+
+    minimumLineSpacing = 20
+    minimumInteritemSpacing = 20
+    sectionInset = NSEdgeInsets(
+      top: 20,
+      left: 40,
+      bottom: 60,
+      right: 40
+    )
+  }
+
+  override func prepare() {
+    guard let collectionView = collectionView
+      else { return }
+
+    let width = collectionView.frame.size.width
```

Comment:
FWIW I'd use the collection view's bounds here, because the layout should be in the coordinate system of the collection view itself.